assets/_custom.scss

```diff
@@ -1,3 +1,27 @@
+/* You can add custom styles here. */
+
+// @import "plugins/numbered";
+// @import "plugins/scrollbars";
+
+:root {
+    --gray-50: #fafbfc;
+    --table-2n-background: var(--gray-100);
+}
+@media(prefers-color-scheme: dark) {
+    :root {
+        --gray-50: rgba(255, 255, 255, 0.05);
+        --table-2n-background: var(--gray-50);
+    }
+}
+
 .container {
-	max-width: 100rem
+    max-width: 100rem;
+}
+
+.markdown {
+    table tr {
+        &:nth-child(2n) {
+            background: var(--table-2n-background);
+        }
+    }
```

Comment:
I have mixed feelings about this. Alternating row colors are really useful on large tables (such as the [title list](https://docs.mikage.app/Title_list/)). However for smaller tables and for sequential structures like IPC command tables, non-alternating colors are the right choice indeed. Is there a way we could opt into alternating row colors for individual tables that benefit from it (without forcing those tables to be HTML)?

Doesn't need to happen now, just curious if we could support both.

Reply:
Actually, just to double-check: Was the intent here indeed to disable alternating row colors, or is this an unintended effect for the light theme?

Reply:
It's not an unintended effect, the default book theme has alternating colors, but as usual with people that have very little design and color theory, both are offset a fixed number of RGB units from the background color, which results in a very different effect between light and dark because of the human eyes' nonlinear sensitivity.

This is just here to fix that, but the point about being opt-in is a good one, although I have no idea on how to make this fit in nicely with Markdown, is there a way to specify table classes in this dialect?

Reply:
> is there a way to specify table classes in this dialect?

Doesn't seem like it, but goldmark has no problem with wrapping the entire markdown table in a `<div>`. So we can just do that and then define a CSS rule for `.withalternatingrows > table > ...` :)